docs/c_cxx/Doxyfile

```diff
@@ -12,6 +12,16 @@
 # For lists, items can also be appended using:
 # TAG += value [value, ...]
 # Values that contain spaces should be placed between quotes (\" \").
+#
```

Comment:
Great work! There is one more file that needs to be upgraded, and that is for C# Docs. It is under tools folder. Perhaps, we could move them together?

Reply:
I'm going to go ahead and merge this one, as the C/C++ docs are currently broken. If you let me know the other files that needs to be updated, I'm happy to do it in a subsequent PR